datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -30,6 +30,7 @@ ldap3==2.5
 lxml==4.5.0
 lz4==2.2.1
 meld3==1.0.2
+mmh3==2.5.1
```

Comment:
Earlier discussions in Slack suggested using [pyfasthash](https://github.com/flier/pyfasthash) instead of [mmh3](https://github.com/hajimes/mmh3). We have a requirement that the hashes computed here match the hashes produced by our go backend. I haven't been able to produce a matching hash with the pyhash API, but I have with mmh3. Also the murmur 3 code in `pyfasthash` hasn't been updated for [7 years](https://github.com/rurban/smhasher/blob/333f102f3c05ecfe45d7e0f41281066bd9b77d8d/MurmurHash3.cpp) (compared to 3 years ago with mmh3).

Reply:
We'll use this after it's fixed https://github.com/explosion/murmurhash/issues/14